packages/core/src/Stores/client-store.js

```diff
@@ -1030,13 +1030,20 @@ export default class ClientStore extends BaseStore {
 
     @action.bound
     fetchStatesList() {
-        WS.statesList({
-            states_list: this.accounts[this.loginid].residence
-        }).then(response => {
-            runInAction(() => {
-                this.states_list = response.states_list || [];
+        return new Promise((resolve, reject) => {
+            WS.statesList({
+                states_list: this.accounts[this.loginid].residence
+            }).then(response => {
+                if (response.err) {
+                    reject(response.error)
```

Comment:
Is `err` a valid object on `response`?  It rejects with `response.error` but is checking for `response.err`?

Reply:
ah thanks should be response.error